src/json/get-decorated-data-path.js

```diff
@@ -29,7 +29,9 @@ function getTypeName(obj) {
   if (!obj || !obj.children) {
     return '';
   }
-  const type = obj.children.filter(child => child.key.value === 'type');
+  const type = obj.children.filter(child => {
+    child && child.key && child.key.value === 'type';
```

Comment:
@torifat the existing logic seems like searching for
```
type: "type"
```
is this correct? or it's a mistake in first place?